src/System.Runtime.Extensions/tests/System/IO/Path.cs

```diff
@@ -61,22 +95,50 @@ public static void GetExtension(string expected, string path)
     [Fact]
     public static void GetFileName()
     {
+        Assert.Equal(null, Path.GetFileName(null));
```

Comment:
Path.Combine can't be used in an attribute, and it didn't seem worth separating out those that use it.